contrib/scripts/localnet-uniswap.sh

```
@@ -0,0 +1,261 @@
+
+# Check if FLAG_SKIP_BUILD was set to true
+if ! $FLAG_SKIP_BUILD; then
+  build_from_source
+fi
+
+echo_info "Features flags:"
+echo "FLAG_SKIP_BUILD: $FLAG_SKIP_BUILD"
+
+SEDOPTION=""
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  SEDOPTION="''"
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)